src/views/portals/outings-stats/OutingsStatsPart.vue

```diff
@@ -108,6 +108,26 @@ const getHikingRatingColor = function (rating, minRating, maxRating) {
   return getRedToGreenColor(rating, common.attributes.hiking_ratings, minRating, maxRating);
 };
 
+const getActivityColor = function (activity) {
+  return {
+    ice_climbing: 'Aqua',
+    snow_ice_mixed: 'LightSteelBlue',
```

Comment:
Why are you using sometimes uppercase, sometimes lowercase?

By the way, I have a personal feeling that predefined colors are not really colors you should use. They are often too flashy. BUt there is a whole topic for these colors on the forum :)

Reply:
> Why are you using sometimes uppercase, sometimes lowercase?

Because sometimes I've handwritten it, sometimes copy-paste it :) I'll make this homogenous

> By the way, I have a personal feeling that predefined colors are not really colors you should use. They are often too flashy. BUt there is a whole topic for these colors on the forum :)

Agree, but I don't have the time to look for the perfect set. We'll start with this set, and let anybody who's willing to try to improve it.